app/src/main/java/com/target/dealbrowserpoc/dealbrowser/DealListFragment.java

```diff
@@ -1,62 +0,0 @@
-package com.target.dealbrowserpoc.dealbrowser;
```

Comment:
In order to preserve history,
java can be renamed to kt and commit and then make actual conversion and make a different commit
haven't done that so history is lost